<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_acceuil">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>&lt;h1&gt;&lt;b&gt;&lt;font color=blue
&gt;Interface homme-machine avanc√©e&lt;/font&gt;&lt;/b&gt;&lt;/h1&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;h3&gt;&lt;b&gt;&lt;font color=blue
&gt;Tp,Mini_projets,Projets&lt;/font&gt;&lt;/b&gt;&lt;/h3&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>&lt;h3&gt;&lt;b&gt;&lt;font color=blue
&gt;Tp&lt;/font&gt;&lt;/b&gt;&lt;/h3&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>1 Example</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Labels" name="view_image">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="mouseTracking">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>&lt;h3&gt;&lt;b&gt;&lt;font color=blue
&gt;Mini_projets&lt;/font&gt;&lt;/b&gt;&lt;/h3&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>0 Example</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Labels" name="view_image_2">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>&lt;h3&gt;&lt;b&gt;&lt;font color=blue
&gt;Projets&lt;/font&gt;&lt;/b&gt;&lt;/h3&gt;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>0 Example</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Labels" name="view_image_3">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_tp">
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>30</y>
          <width>81</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;h1&gt;&lt;b&gt;&lt;font color=blue
&gt;Tp&lt;/font&gt;&lt;/b&gt;&lt;/h1&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>200</y>
          <width>526</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Faire une representation graphique des donne√©s en diagramme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Labels" name="run_tp1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>&lt;h4&gt;&lt;b&gt;&lt;font color=blue
&gt;Run&lt;/font&gt;&lt;/b&gt;&lt;/h4&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>112</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_miniProjet">
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>30</y>
          <width>301</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h1&gt;&lt;b&gt;&lt;font color=blue
&gt;Mini_projets&lt;/font&gt;&lt;/b&gt;&lt;/h1&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>112</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_projet">
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>321</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h1&gt;&lt;b&gt;&lt;font color=blue
&gt;Projets&lt;/font&gt;&lt;/b&gt;&lt;/h1&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>112</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Labels</class>
   <extends>QLabel</extends>
   <header location="global">labels.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
